registry.json

```diff
@@ -5419,5 +5419,19 @@
    "telegram": "Dinoaus73",
    "operatorName": "John Dean",
    "github": ""
+ },
+ {
+   "ticker": "LIT",
+   "poolId": "02c7fc9aa1f415c8596f0957f68a3908dbdf458da4d09be93ddd608b",
+   "social": {
+     "twitter": "LitStaking",
+     "website": "https://litstaking.com",
+     "facebook": "",  
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "",
```

Comment:
You must provide a telegram handle and it must be that of an individual and not a group or channel

Reply:
Added the Telegram handle. Thank you!